src/test/java/com/accessibility/integration/test/WebDriverProvider.java

```diff
@@ -7,7 +7,6 @@
 
 public class WebDriverProvider implements Provider<WebDriver> {
 
-	@Override
```

Comment:
If we don't remove that, the following error will be show:
The method get() of type WebDriverProvider must override a superclass
method	WebDriverProvider.java	/webdriver-accessibility/src/test/java/com/accessibility/integration/test	line
11	Java Problem

The quickfix is to remove the @Override annotation.
>> @@ -7,7 +7,6 @@
>>
>>  public class WebDriverProvider implements Provider<WebDriver> {
>>
>> -	@Override
>
> Any Reason we removed these?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/nikulkarni/webdriver-accessibility/pull/3/files#r53662694